src/main/java/net/snowflake/ingest/streaming/internal/DropChannelRequestInternal.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2024 Snowflake Computing Inc. All rights reserved.
+ */
+
+package net.snowflake.ingest.streaming.internal;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import net.snowflake.ingest.streaming.DropChannelRequest;
+import net.snowflake.ingest.utils.Utils;
+
+/** Class used to serialize the {@link DropChannelRequest} */
```

Comment:
should this (and OpenChannelRequestInternal) really be public classes?

Reply:
Change to package private.